pkg/kademlia/kademlia.go

```diff
@@ -145,14 +145,15 @@ func (k *Kademlia) DumpNodes(ctx context.Context) (_ []*pb.Node, err error) {
 func (k *Kademlia) Bootstrap(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
 	defer k.bootstrapFinished.Release()
+	defer k.timeTrack(time.Now(), "Bootstrap")
```

Comment:
There's no need to add the extra "Bootstrap" argument since it's used only once here.